libs/common/src/response-messages/index.ts

```diff
@@ -424,7 +426,11 @@ export const ResponseMessages = {
             invalidMessage: 'Invalid transaction details. Missing "message" property.',
             invalidTransactionMessage: 'Invalid transaction details',
             ecosystemRoleNotMatch: 'Ecosystem role not match',
-            orgEcoIdRequired: 'OrgId & EcosystemId is required'
+            orgEcoIdRequired: 'OrgId & EcosystemId is required',
+            ecosystemMembersNotExists: 'Ecosystem members does not exists',
+            notAbleToDeleteEcosystem: 'You cannot delete the ecosystem, because you are the ecosystem lead',
+            ecosystemNotExists: 'Ecosystem does not exists',
+            removeEcosystemData: ', first you have to remove ecosystem data'
```

Comment:
Can we remove the ',' since we can reuse this message elsewhere

Reply:
Resolved